specification/paths/CalculateLandedCosts.json

```diff
@@ -35,26 +35,28 @@
                     "properties": {
                       "attributes": {
                         "required": [
-                          "recipient_address",
-                          "sender_address",
-                          "physical_properties",
```

Comment:
I think we should remove the `allOf` with the `Shipment` component and make this a separate data object.